ttnn/cpp/ttnn/operations/moreh/moreh_arange/moreh_arange_pybind.cpp

```diff
@@ -0,0 +1,27 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "moreh_arange_pybind.hpp"
+
+#include "pybind11/decorators.hpp"
+#include "ttnn/operations/moreh/moreh_arange/moreh_arange.hpp"
+
+namespace ttnn::operations::moreh::moreh_arange {
+void bind_moreh_arange_operation(py::module& module) {
+    bind_registered_operation(
+        module,
+        ttnn::moreh_arange,
+        "Moreh Arange Operation",
+        ttnn::pybind_arguments_t{
+            py::arg("start"),
+            py::arg("end"),
+            py::arg("step"),
+            py::arg("any"),
+            py::arg("output_tensor") = std::nullopt,
+            py::arg("untilize_out") = false,
+            py::arg("output_dtype") = std::nullopt,
```

Comment:
In ttnn we use kw_only arguments dtype and memory_config. 
please adjust for consistency. 
Also make sure that arguments which are kw args in torch are also kw args here. 

Thank you!